tests/unittests/OnnxImporterTest.cpp

```diff
@@ -3642,6 +3641,67 @@ TEST(onnx, importSign) {
   importUnary(netFilename, input, inputShape, outputShape, expectedValues);
 }
 
+static void importCompare(const std::string &netFilename,
+                          llvm::ArrayRef<float> LHS, llvm::ArrayRef<float> RHS,
+                          llvm::ArrayRef<dim_t> LHSShape,
+                          llvm::ArrayRef<dim_t> RHSShape,
+                          llvm::ArrayRef<dim_t> outputShape,
+                          llvm::ArrayRef<float> expectedValues) {
+  float delta = 1e-08;
+  ExecutionEngine EE{};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+  PlaceholderBindings bindings;
+  Placeholder *graphOutputVar;
+  // Load the .onnxtxt model.
+  Type LHSType(ElemKind::FloatTy, LHSShape);
+  Type RHSType(ElemKind::FloatTy, RHSShape);
+  ONNXModelLoader onnxLD(netFilename, {"LHS", "RHS"}, {&LHSType, &RHSType}, *F);
+  graphOutputVar = EXIT_ON_ERR(onnxLD.getSingleOutput());
+  auto LHSPH = mod.getPlaceholderByNameSlow("LHS");
```

Comment:
nit: `auto *LHSPH` (if you copy/pasted this from elsewhere feel free to fix the other uses too 😄 )